ts/features/bonusVacanze/components/AvailableBonusItem.tsx

```diff
@@ -1,21 +1,16 @@
-import { fromNullable } from "fp-ts/lib/Option";
-import * as pot from "italia-ts-commons/lib/pot";
 import { Badge, Grid, ListItem, Row, Text, View } from "native-base";
 import * as React from "react";
 import { Image, Platform, StyleSheet } from "react-native";
-import { connect } from "react-redux";
+import { BonusAvailable } from "../../../../definitions/content/BonusAvailable";
 import I18n from "../../../i18n";
-import { serviceByIdSelector } from "../../../store/reducers/entities/services/servicesById";
-import { GlobalState } from "../../../store/reducers/types";
 import variables from "../../../theme/variables";
-import { BonusAvailable } from "../types/bonusesAvailable";
 
 type OwnProps = {
```

Comment:
Maybe now that we removed the mapStateToProps it is useless to have `OwnProps` and `Props` and take only one of the 2 what do you think?

Reply:
done in https://github.com/pagopa/io-app/pull/1896/commits/ed5bf6a2e7697c751ce9c00a2de81349af857846